articles/virtual-machines/linux/disk-encryption.md

```diff
@@ -68,6 +68,7 @@ For now, customer-managed keys have the following restrictions:
 - All resources related to your customer-managed keys (Azure Key Vaults, disk encryption sets, VMs, disks, and snapshots) must be in the same subscription and region.
 - Disks, snapshots, and images encrypted with customer-managed keys cannot move to another subscription.
 - If you use the Azure portal to create your disk encryption set, you cannot use snapshots for now.
+- Virtual machines encrypted using encryption at rest with customer managed keys cannot be encrypted using Azure Disk Encryption.
```

Comment:
Would you adjust this a bit first?

"Managed disks encrypted using customer-managed keys cannot also be encrypted with Azure Disk Encryption."